src/main/java/org/opentripplanner/middleware/spark/Main.java

```diff
@@ -26,7 +28,7 @@
 
 public class Main {
     private static final Logger LOG = LoggerFactory.getLogger(Main.class);
-    private static final String DEFAULT_ENV = "configurations/default/env.yml";
+    private static final String DEFAULT_ENV = "configurations/default/env.yml.tmp";
```

Comment:
No, this should remain `/env.yml`. (People should be copying the `yml.tmp` file into their own `.yml file` and placve their info in that file.)